code/index.html

```diff
@@ -1,30 +1,113 @@
+        aspernatur sed dignissimos? Cum doloremque architecto optio voluptatem aliquam voluptatum quaerat atque,
+        mollitia eos recusandae.</p>
+      <a href="" class="menu-button"> Find out more</a>
+    </div>
+    </div>
+
+  </header>
+  <!-- Signup form -->
+  <section class="grid-container">
+
+    <div id="signup" class="signup">
+      <form action="http://httpbin.org/anything" method="POST">
+        <!--Lable, input-->
+        <fieldset>
+          <legend>
+            <h2 class="sign-up-text"> Sign Up </h2>
+          </legend>
+          <label class="form-text" for="Name">First Name</label>
+          <input type="text" name="fName" id="fName" placeholder="Required" required>
+          <br><br>
+          <label class="form-text" for="lName">Last Name</label>
+          <input type="text" name="lName" id="lName" placeholder="Required" required>
+          <br><br>
+          <label class="form-text" for="email">Email adress</label>
+          <input type="email" name="email" id="email" placeholder="Required" required>
+          <br><br>
+          <input class="checkbox" type="checkbox" name="newsletter" id="checkbox">
+          <label for="newsletter"> I want to recieve news</label>
+          <br><br>
+          <button class="form-button" type="submit" type="submit" value="Send">Send</button>
+        </fieldset>
+      </form>
+    </div>
+
+
+    <div class="accordion">
+      <!-- Frequently Asked Questions Accordion -->
+      <fieldset>
+        <legend>
+          <h2 class="faq-title"> Frequently Asked Questions </h2>
+        </legend>
+
+        <!--<div class="accordion">-->
+        <button type="button" class="accordion__button">What does a Hajk-ing tour cost?</button>
+
+        <div class="accordion__content">
+          <p>
+            Pengar..
+          </p>
+        </div>
+
+        <button type="button" class="accordion__button">What clothes are recommended for the Hajk?</button>
+
+        <div class="accordion__content">
+          <p>
+            Clothes after weather. "There are no such thing as bad weather, just bad clothes" Comfy shoes.
+          </p>
+        </div>
+
+        <button type="button" class="accordion__button">Do I need to bring my own food?</button>
+
+        <div class="accordion__content">
+          <p>
+            Ther will be stops along the way to either buy pre-made food or to cook what you brought yourself.
+          </p>
+        </div>
+
+        <button type="button" class="accordion__button">Will this be a guided Hajk-tour?</button>
+
+        <div class="accordion__content">
+          <p>
+            A guide comes as extra charge. But it is recommended for beginner-Hajkers.
+          </p>
+        </div>
+
+    </div>
+
+    </fieldset>
+    </div>
+  </section>
+
+  <footer>
+    <div class="footer-signature">
+      <h1>Hajker guide </h1>
+      <p>Eddie Lundgren</p>
+    </div>
+  </footer>
+  <script src="script.js"></script>
+</body>
+
+</html>
```

Comment:
Learned that you need a newline at the end of file. To avoid that little red alert.